example/fundamentals/library-deps/bom-2-dependency-management/build.mill

```diff
@@ -0,0 +1,54 @@
+// Pass dependencies to `dependencyManagement` in a `JavaModule` / `ScalaModule` / `KotlinModule`, like
+
+//// SNIPPET:BUILD1
+package build
+import mill._, javalib._
+
+object dependencyManagement extends JavaModule {
```

Comment:
Let's rename this `foo` or something, otherwise the name is confusing when duplicated below in `def dependencyManagement`. The code and english explanation should be enough to describe what the example is about, and we can add a `== subheading` if necessary